Python/Python.sublime-syntax

```diff
@@ -234,6 +234,9 @@ contexts:
           scope: invalid.illegal.missing-in.python
           pop: true
         - include: target-list
+    - match: \bcase\b
```

Comment:
Until 3.10, `case` is a normal identifier like any other. In order not to regress these cases, we will need to use branching to highlight `case` as a keyword only in match-case blocks. It doesn't need to be perfect, but it should be reasonable.